docs/installation-and-operations/configuration/environment/README.md

```diff
@@ -61,82 +67,253 @@ Please note: The Configuration is a HashWithIndifferentAccess and thus it should
 
 # Supported environment variables
 
-Below is the full list of supported environment variables that can be used to override the default configuration of your OpenProject installation:
+A list of all configuration variables that can be overridden by defining them in the environment from bash could be generated by executing the following rake task:
+
+For packaged installations:
+
+```bash
+sudo openproject run bundle exec rake 'setting:available_envs'
```

Comment:
You don't need the quotes around `setting:available_envs`.

Reply:
was corrected